main.go

```diff
@@ -51,6 +51,8 @@ type GlobalFlags struct {
 	EnableIndexStats       bool `name:"collector.indexstats" help:"Enable collecting metrics from $indexStats"`
 	EnableCollStats        bool `name:"collector.collstats" help:"Enable collecting metrics from $collStats"`
 
+	EnableOverrideDescendingIndex bool `name:"metrics.overridedescendingindex" help:"Enable descending index name override to replace -1 with _DESC"`
```

Comment:
I think this should be enabled by default and the parameter name should be something like "disable_index_renaming" or something similar.
Since without this feature we could break the exporter, the default behavior should be to prevent errors.
@ShashankSinha252 , @rnovikovP what do you think?

Reply:
I think it's best to not break compatibility in our 0.x releases. 1.0 release would be a good time for us to go ahead and break compatibility. It would lead to a smooth experience for users.